circle.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - pip install scipy
     - pip install scikit-learn
     - pip install nose-timer
+    - pip install memory_profiler psutil
```

Comment:
Yeah, I just wanted to compare the results across CI's (as the tests fail on travis). I'll revert this.